server/routes/induction/common/checkYourAnswersController.ts

```diff
@@ -7,6 +7,16 @@ import { buildNewPageFlowHistory } from '../../pageFlowHistory'
  * Abstract controller class defining functionality common to both the Create and Update Induction journeys.
  */
 export default abstract class CheckYourAnswersController extends InductionController {
+  getBackLinkUrl(_req: Request): string {
```

Comment:
The Induction page superclass defines the abstract methods `getBackLinkUrl` and `getBackLinkAriaText` , and all implementations needed to implement these methods (which was then in turn used in the nunjucks templates to render the "back" links)

Anyway, once we have migrated all Induction pages to use JS back links, these methods can be nuked from the superclass and implementations 👍  (I'm looking forward to that PR 😁 )